app/src/main/java/com/somestupidappproject/betago/utils/ScoringUtils.java

```diff
@@ -0,0 +1,75 @@
+package com.somestupidappproject.betago.utils;
+
+import com.somestupidappproject.betago.board.Board;
+import com.somestupidappproject.betago.board.Stone;
+import com.somestupidappproject.betago.moves.Move;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Stack;
+
+/**
+ * Created by sapatel on 7/14/2017.
+ */
+
+public class ScoringUtils {
+    private static final String TAG = "betago.EndGameUtils";
+
+    /**
+     * returns the player with the highest score.
+     * This is not efficient but can optimize if necessary
+     * @param board
+     * @param previousMoves
+     * @return
+     *  returns 1 if black wins
+     *  returns 2 if white wins
+     *  returns -1 if tie
+     */
+    public static int getWinningPlayer(Board board, Stack<Move> previousMoves) {
+        ScoreCount captureScores = getCaptureScores(previousMoves);
+        int blackScore = captureScores.getBlackScore();
+        int whiteScore = captureScores.getWhiteScore();
+
+        for (int i = 0; i < board.getBoardSize(); i++) {
+            for (int j = 0; j < board.getBoardSize(); j++) {
+                if (board.getStone(i, j).getColor() == 0) {
+                    int territoryOwner = LogicUtil.getTerritoryOwner(board, board.getStone(i, j));
+                    if (territoryOwner == 1) {
+                        blackScore++;
+                    } else if (territoryOwner == 2) {
+                        whiteScore++;
+                    }
+                }
+            }
+        }
+
+        if (blackScore > whiteScore) {
+            return Stone.BLACK;
+        } else if (whiteScore > blackScore) {
+            return Stone.WHITE;
+        }
+        return -1;
```

Comment:
yeah I should just return a string of either BLACK, WHITE, or TIE as returning a player doesn't make much sense either (returning both players in the event of a tie seems bad too)